provider/aws/sia-eks/authn.go

```diff
@@ -31,7 +31,7 @@ func GetEKSPodId() string {
 	return podId
 }
 
-func GetEKSConfig(configFile, metaEndpoint string, useRegionalSTS bool, region string) (*options.Config, *options.ConfigAccount, error) {
+func GetEKSConfig(profileConfigFile, configFile, metaEndpoint string, useRegionalSTS bool, region string) (*options.Config, *options.ConfigAccount, *options.AccessProfileConfig, error) {
```

Comment:
in EC2 Config we have configFile, profileConfigFile - so let's keep the order consistent